api/helpers/add-game-to-match.js

```diff
@@ -0,0 +1,68 @@
+module.exports = {
+  friendlyName: 'Add Game to Match',
+
+  description:
+    'Adds the provided game to the appropriate match. Will create a match if not found and will end the match if this game finishes it.',
+
+  inputs: {
+    game: {
+      type: 'ref',
+      description:
+        'Game record to be added to the appropriate match. Should include an array of players',
+      required: true,
+    },
+  },
+  fn: async ({ game }, exits) => {
+    try {
+      const [player1, player2] = game.players;
+      let relevantMatch = await sails.helpers.findOrCreateCurrentMatch(player1.id, player2.id);
+      if (!relevantMatch) {
+        return exits.error('Could not add game to match');
+      }
+      if (relevantMatch.endTime || relevantMatch.winner) {
+        return exits.success(null);
```

Comment:
Why pass null?

Reply:
No good reason--this is half-baked. This is the case where the match is already over and we shouldn't add the new game to the match so it returns early. Could maybe skip null and just `return exits.success()` or even make a specific exit for this case so it can be handled more particularly by the consumer of this method